### PR TITLE
Feature/infopages

### PR DESCRIPTION
**Pull request by ctychen**
footer, info pages and links

### CODE REVIEW

File: templates/about_us.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+{% load static %}
+{% block extra_css %}
+{% endblock extra_css %}
```
Comment (chenyx512) [inline templates/about_us.html]: you don't need this tag if you  are not using extra css

File: views.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+"""
+This view serves only the basic website structure like index page
+"""
+from django.shortcuts import render
+
+
+def index(request):
+    return render(request, 'index.html')
+def about_us(request):
```
Comment (chenyx512) [inline views.py]: you should stick to pip8 and leave 2 blank lines between functions

**Review by chenyx512 — CHANGES_REQUESTED** [review]
For the 2 backend files, add 2 blank lines between functions in jiezi/views.py.
Wait for @yu-george and @Miclin1024 to review front end files

File: views.py (hunk 2 above)
Comment (georgeyjm) [reply views.py]: > you should stick to pip8 and leave 2 blank lines between functions

He means PEP8.

File: templates/base.html [hunk 3]
```diff
@@ -68,13 +69,32 @@
...
-
-<footer class="full-width-bg-container" id="footer-bg">
-  <div class="main-content-container" id="footer">
-    <p>SolvedChinese.org&#169;2019</p>
-    <a href="https://github.com/chenyx512/Jiezi" target="_blank"><img src="/static/images/github.svg" class="footer-icon"></a>
-  </div>
-</footer>
+<div class="footer" style="background: #ccc">
+        <a href="{% url 'index' %}">
+            <img src="{% static "images/base/logo.png" %}" style=" margin-left: 12vw; padding-left:3vw; height:4vw; vertical-align:middle">
```
Comment (georgeyjm) [inline templates/base.html]: A couple of things:
- We are using the new logo now, it's [this file](https://github.com/chenyx512/jiezi/blob/master/static/images/logo-color-full.svg), please use the new logo.
- Indent HTML files with 2 spaces, and align the tags.
- Generally, a repeated style (e.g. `vertical-align: middle`) can be changed into a class, this is to reduce code redundancy and increase readability.
- For the mail icon, use font-awesome (we have Pro), not an image.

File: templates/help.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load static %}
+{% block extra_css %}
+{% endblock extra_css %}
+{% block title %}Help{% endblock title %}
```
Comment (georgeyjm) [inline templates/help.html]: Just out of curiosity, should the title be "Help" or something like "Help | Solved"?
@chenyx512 you got any preferences we should stick to?

File: templates/help.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load static %}
+{% block extra_css %}
+{% endblock extra_css %}
```
Comment (georgeyjm) [inline templates/help.html]: As aforementioned by Jason, if a block is empty, don't bother adding the block tags.

File: views.py (hunk 2 above)
Comment (chenyx512) [reply views.py]: yes

**Review by georgeyjm — CHANGES_REQUESTED** [review]
First of all, thank you for your work!
There are several changes regarding the front-end code that I would like, especially those concerning code styles, we should be consistent in styles.

File: templates/help.html (hunk 4 above)
Comment (chenyx512) [reply templates/help.html]: That is your choice, or you could ask Jenny for design specs, the "Help | Solved" seems to be @Miclin1024 's style

Comment (georgeyjm) [reply templates/help.html]: I think we ought to be consistent with this.
I will consult Jenny.

File: urls.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+    # front-page urls
+    path('index/', views.index, name="index"),
+    path('', RedirectView.as_view(url='index')),
+    path('about_us/', views.about_us, name="about_us"),
+    path('the_science_behind/', views.the_science_behind, name="the_science_behind"),
+    path('help/', views.help, name="help"),
+]
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+urlpatterns += staticfiles_urlpatterns()
```
Comment (Miclin1024) [inline urls.py]: You don't need a new urls.py to add the three paths. For example, if you wish the access point to be `localhost:8000/about_us/`, the lines should be added to `jiezi/url.py`.

File: views.py [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    return render(request, 'about_us.html')
+def the_science_behind(request):
+    return render(request, 'the_science_behind.html')
+def help(request):
+    return render(request, 'help.html')
```
Comment (Miclin1024) [inline views.py]: Same as `urls.py`, the view functions should be added to `jiezi/views.py`.

File: templates/about_us.html [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+                <br><br>
+<div style="border-left: 50px solid #ffff;">
+                Solved incorporates ancient wisdom with modern technology. It is research-based, individualized, engaging, user-friendly, and free! <a href="{% url 'the_science_behind' %}">Read more about the art and science behind it here.</a></div>
+                <br><br>
+<div style="color:#B62C25; border-left: 50px solid #ffff; font-size:2vw"> The Team</div><div style="border-left: 50px solid #ffff;"><br>
+                We are a group of Chinese high schoolers passionate about linguistics and education. We attend schools in China, the U.S., Costa Rica, and Singapore.</div>
+                <br><br><div style="border-left: 50px solid #ffff;">
+                We’d love to hear any questions or comments, and we are seeking more volunteers to help improve and expand this project. Please feel free to contact us at <a href="mailto:contact@solvedchinese.org">contact@solvedchinese.org</a></div>
+
```
Comment (Miclin1024) [inline templates/about_us.html]: I would say keeping the styling in css stylesheet, for both styling consistency and code reuse.

**Review by Miclin1024 — CHANGES_REQUESTED** [review]

**Comment by chenyx512** [comment]
I am now taking over this PR

**Comment by chenyx512** [comment]
Problems now:
The button bar now looks very ugly and needs to be changed. Moreover, it is no longer staying at the bottom is the page is shorter.
The syntax problem in html is not fixed still.
The about us/ the science behind pages need confirmation from the ling team as well.
@yu-george @Miclin1024 @carlguo866 Anyone of you want to jump in?

**Comment by chenyx512** [comment]
@yu-george @carlguo866 Any of you want to continue working on this PR?

**Pull request merged**